apollo-compatibility/src/main/scala/Main.scala

```diff
@@ -0,0 +1,35 @@
+import caliban.CalibanError
```

Comment:
Can you sort imports?

Reply:
I forgot to push the commit with this change, I'll do another with just the import formatting